sipnet.c

```diff
@@ -48,7 +48,7 @@
 // Lloyd-Taylor model is R = R0 * e^(E0 * (1/(-T0) - 1/(T - T0))) (see Lloyd and Taylor 1994, "On the temperature dependence of soil respiration")
 // where R0 is rate at 0 degrees C, T is soil temp., E0 and T0 are parameters
 
-#define SEASONAL_R_SOIL 0 && !LLOYD_TAYLOR
+#define SEASONAL_R_SOIL 1 && !LLOYD_TAYLOR
```

Comment:
Not sure why the addition of the leaf pool requires changing the definition of this variable

Reply:
It doesn't hehe

I was testing with this flag myself, realized that the compiler gives errors due to missing parenthesis, thought I'd just add that fix as well. I can turn it back off